pkg/varnishcontroller/controller/endpoints.go

```diff
@@ -154,6 +159,12 @@ func (r *ReconcileVarnish) getPodsInfo(ctx context.Context, namespaces []string,
 				}
 			}
 		}
+
+		if !portFound {
+			errMsg := fmt.Sprintf("Backend pod %s/%s ignored since none of it's containers have port %q defined", pod.Namespace, pod.Name, validPort.String())
```

Comment:
```suggestion
			errMsg := fmt.Sprintf("Backend pod %s/%s ignored since none of its containers have port %q defined", pod.Namespace, pod.Name, validPort.String())
```